dev/com.ibm.websphere.appserver.features/visibility/public/audit-1.0/resources/l10n/com.ibm.websphere.appserver.audit-1.0.properties

```diff
@@ -15,19 +15,33 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=The Liberty Audit feature is used to report and track auditable events to ensure the integrity of your system. \
-The Liberty Audit feature introduces an infrastructure which serves two purposes: \
- - Confirming the effectiveness and integrity of the existing configuration \
- - Identifying areas where improvement to the configuration may be needed  \
-The Liberty Audit feature has the ability to capture the following auditable events: \
- - Basic authentication \
- - Start and stop of the Audit service \
- - Form login \
- - Client certificate login \
- - Servlet runAs delegation \
- - Form logout \
- - Servlet 3.0 login, logout and authenticate APIs \
-The Liberty Audit feature supports the Cloud Auditing Data Federation (CADF) event model.  The CADF model describes a data model and associated schema definitions for an audit event. \
-The feature provides a default implementation, the AuditFileHandler, which emits human-readable audit records to a file-based log.   Each audit record is emitted in JSON format. \
-
-
+description=The Liberty Audit feature is used to report and track auditable events to ensure the integrity of your system. \n\
+The Liberty Audit feature introduces an infrastructure which serves two purposes: \n\n\
+ - Confirming the effectiveness and integrity of the existing configuration \n\
+ - Identifying areas where improvement to the configuration may be needed  \n\n\
+The Liberty Audit feature has the ability to capture the following auditable events: \n\n\
+ - Basic authentication \n\
+ - Start and stop of the Audit service \n\
+ - Form login authentication \n\
+ - Client certificate authentication \n\
+ - Servlet runAs delegation \n\
+ - Failover to basic authentication \n\
+ - Unprotected servlet authorization \n\
+ - Servlet 3.0 APIs: login/logout/authenticate \n\
+ - JACC web authorization \n\
+ - Form logout \n\
+ - JACC EJB authorization \n\
+ - EJB delegation \n\
+ - SCIM operations/member management \n\
+ - Dynamic audit feature handling \n\
+ - EJB authorization \n\
+ - JMX MBean operations \n\
+ - JMX Notifications \n\
+ - JMX MBean registration \n\
+ - JMX MBean attribute operations \n\
+ - JMS Authentication \n\
+ - JMS Authorization \n\
+ - OAuth application password and token management \n\
+ - SAF authorization \n\n\
+The Liberty Audit feature supports the Cloud Auditing Data Federation (CADF) event model.  The CADF model describes a data model and associated schema definitions for an audit event. \n\n\
+The feature provides a default implementation, the AuditFileHandler, which emits human-readable audit records to a file-based log.   Each audit record is emitted in JSON format.
```

Comment:
Only 1 space between "(CADF) event model." and "The CADF model", and only one space between "file-based log." and "Each audit record"

Reply:
Overall question:  All of this text was reviewed by IDD a year and half ago, and what is there is based on IDD's feedback then.  If none of the text is being changed, why is it getting re-reviewed?    Different person, different feedback, I get it.   But we're not changing text here, just format.

Reply:
@emilytee LibbyBot flagged it as requiring an ID review, so I reviewed what was changed. Sometimes we get a different pair of eyes on something and pick up things that might've been overlooked. All of the changes I suggested are minor.